src/components/medication-dispense-review.component.tsx

```diff
@@ -14,30 +14,135 @@ import {
   const [isEditingFormulation, setIsEditingFormulation] = useState(false);
+  const [isSubstitution, setIsSubstitution] = useState(false);
+  // Dosing Unit eg Tablets
+  const [drugDosingUnits, setDrugDosingUnits] = useState([]);
+  // Dispensing Unit eg Tablets
+  const [drugDispensingUnits, setDrugDispensingUnits] = useState([]);
+  // Route eg Oral
+  const [drugRoutes, setDrugRoutes] = useState([]);
+  // Frequency eg Twice daily
+  const [orderFrequencies, setOrderFrequencies] = useState([]);
+  // type of substitution question
+  const [substitutionTypes, setSubstitutionTypes] = useState([]);
+  // reason for substitution question
+  const [substitutionReasons, setSubstitutionReasons] = useState([]);
+
   const isTablet = useLayoutType() === "tablet";
 
+  const { orderConfigObject } = useOrderConfig();
+  const { substitutionTypeValueSet } = useSubstitutionTypeValueSet(
+    config.substitutionType.uuid
+  );
+  const { substitutionReasonValueSet } = useSubstitutionReasonValueSet(
+    config.substitutionReason.uuid
+  );
+
+  useEffect(() => {
+    if (orderConfigObject) {
+      // sync drug route options order config
+      const availableRoutes = drugRoutes.map((x) => x.id);
+      const otherRouteOptions = [];
+      orderConfigObject.drugRoutes.forEach(
+        (x) =>
+          availableRoutes.includes(x.uuid) ||
+          otherRouteOptions.push({ id: x.uuid, text: x.display })
+      );
+      setDrugRoutes([...drugRoutes, ...otherRouteOptions]);
+
+      // sync dosage.unit options with what's defined in the order config
+      const availableDosingUnits = drugDosingUnits.map((x) => x.id);
+      const otherDosingUnits = [];
+      orderConfigObject.drugDosingUnits.forEach(
+        (x) =>
+          availableDosingUnits.includes(x.uuid) ||
+          otherDosingUnits.push({ id: x.uuid, text: x.display })
+      );
+      setDrugDosingUnits([...drugDosingUnits, ...otherDosingUnits]);
+
+      // sync dispensing unit options with what's defined in the order config
+      const availableDispensingUnits = drugDispensingUnits.map((x) => x.id);
+      const otherDispensingUnits = [];
+      orderConfigObject.drugDispensingUnits.forEach(
+        (x) =>
+          availableDispensingUnits.includes(x.uuid) ||
+          otherDispensingUnits.push({ id: x.uuid, text: x.display })
+      );
+      setDrugDispensingUnits([...drugDispensingUnits, ...otherDispensingUnits]);
+
+      // sync order frequency options with order config
+      const availableFrequencies = orderFrequencies.map((x) => x.id);
+      const otherFrequencyOptions = [];
+      orderConfigObject.orderFrequencies.forEach(
+        (x) =>
+          availableFrequencies.includes(x.uuid) ||
+          otherFrequencyOptions.push({ id: x.uuid, text: x.display })
+      );
+      setOrderFrequencies([...orderFrequencies, ...otherFrequencyOptions]);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [orderConfigObject]);
+
```

Comment:
Above effects were just moved from dispense-form, new effects to load substitution stuff below.